source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,137 @@
 WSL 配置指南
 ============
 
-.. note::
+:本节贡献者: |赵志远|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-01-17
+:预计花费时间: 120 分钟
+
+WSL 是什么?
+-------------
+
+Windows Subsystem for Linux（简称 WSL）是一个在 Windows 10 上能够运行原生
+Linux 二进制可执行文件（ ELF 格式）的兼容层，也被称为“适用于 Linux 的 Windows 子系统”。
+
+简单来说，在 Windows 10 上启用 WSL 功能后，就可以在 Windows 上安装 Linux 发行版，
+原样运行 Linux 下的大多数命令行工具、实用工具和应用程序。对于不想安装双系统或者虚拟机的
+用户来说是个不错的选择。
+
+WSL 可以同时安装多个不同的 Linux 发行版，并且随时进行切换，支持利用 ``wsl --export`` 
+命令将整个子系统随时打包备份，并在之后利用 ``wsl --import`` 命令进行还原。与虚拟机相比，
+在启动速度、消耗资源以及性能利用上有较大优势。
+
+WSL 分为 WSL1 和 WSL2 两个发行版本。由于底层原理不同，在大多数情况下，建议使用 WSL2，
+因为它提供更快的性能和 100% 的系统调用兼容性。仅当你的项目需要经常从 Linux 上访问存储
+在 Windows 系统上的文件或者利用 Windows 软件对 Linux 上的文件进行处理时，通过 WSL1 可实现更快的性能。
+
+官方目前没有弃用 WSL1 的计划，并且支持将任何一个已经安装的 Linux 发行版更改为 WSL1 或者 WSL2。
+
+安装 WSL
+---------------
+
+WSL 的安装方式可以参考官方文档：
+
+- 中文指南：https://docs.microsoft.com/zh-cn/windows/wsl/install-win10
+- 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+官方文档对 WSL 安装配置中可能遇到的常见问题都给出了详细的解决方案，请认真阅读。
+
+WSL 可以安装不同的 Linux 发行版，但目前官方并未提供 Fedora 发行版的安装。
+推荐使用  WSL 安装 Ubuntu 20.04 LTS 或 Ubuntu 20.10。
+配置 Linux
+-----------
+
+按照官方文档安装 WSL 之后，就可以根据你的 Linux 发行版进行配置使用了。
 
-   尚未完成。
+如果你安装了 Ubuntu，可以参考 《\ :doc:`/computer/ubuntu-setup` 》进行配置。
 
-   WSL 的安装方式可以参考：
+图形界面显示
+--------------
 
-   - 中文指南：https://blog.walterlv.com/post/how-to-install-wsl2.html
-   - 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+由于 WSL 本身不支持图形界面，我们需要额外安装 X Server 并指定图形输出位置，使得带有图形界面的
+桌面程序可以被显示和运行。
```

Comment:
```suggestion
与 macOS 一样，WLS 也需要安装 X11 来支持图形界面，使得带有图形界面的
应用程序可以正常地显示和运行。

```

Reply:
I don't think it's good to mention macOS here.